docs/workflows/api-token.md

```diff
@@ -397,3 +431,35 @@ func main() {
 
 </TabItem>
 </Tabs>
+
+## Validating Access Tokens:
+
+There are two types of tokens that can be validated: `self-contained` or `referential`. When validating self-contained tokens, consider if revoked tokens are accepted. If revoked tokens are accepted, then the token can be validated [offline](api-token#offline-validation) by validating the signature and parsing the claims of the JWT token. In all other cases the token should be [introspected](api-token#introspection-online-validation).
+
+### Introspection (Online Validation)
+
+A successful introspect query will return information about the token. This endpoint is compliant with [RFC-7662](https://www.rfc-editor.org/rfc/rfc7662). After recieving information about the token, validate the following:
+
+1. Check that either the application id or resource server id is listed in the `aud` of the token. If at least one of the allowed audiences is in the token `aud` claim, then the token is granted.
```

Comment:
`resource server id` should be changed to `resource server identifier`. I agree that this is confusing, but `id` and `identifier` are two different fields of a resource server. ID is used for persistence, whereas identifier will be minted into the `aud` claim of the JWT. Our idea is, that developers may choose whatever string they want to choose for the identifier. Some people suggested this could be a URL.